AdyenActions/UI/View Controllers/QR Code/QRCodeView.swift

```diff
@@ -0,0 +1,118 @@
+    
+    private let model: Model
+    
+    /// The delegate of the view
+    internal weak var delegate: QRCodeViewDelegate?
+    
+    // :nodoc:
+    public var localizationParameters: LocalizationParameters?
+    
+    internal init(model: Model) {
+        self.model = model
+        super.init(frame: .zero)
+        buildUI()
+        backgroundColor = model.style.backgroundColor
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func buildUI() {
+        addLogo()
+        addInstructionLabel()
+        addCopyButton()
+    }
+    
+    internal override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        copyButton.adyen.round(using: model.style.copyButton.cornerRounding)
+    }
+    
+    private func addInstructionLabel() {
+        addSubview(instructionLabel)
+        instructionLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            instructionLabel.topAnchor.constraint(equalTo: logo.bottomAnchor, constant: 24),
+            instructionLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 37),
+            instructionLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -37.0)
+        ])
+    }
+    
+    private func addLogo() {
+        addSubview(logo)
+        logo.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            logo.bottomAnchor.constraint(equalTo: topAnchor, constant: 10),
+            logo.centerXAnchor.constraint(equalTo: centerXAnchor)
+        ])
+    }
+    
+    private func addCopyButton() {
+        addSubview(copyButton)
+        copyButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            copyButton.topAnchor.constraint(equalTo: instructionLabel.bottomAnchor, constant: 29),
+            copyButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 16),
+            copyButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -16),
+            copyButton.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -18)
+        ])
+    }
+    
+    private lazy var copyButton: SubmitButton = {
+        let accessiblityIdentifier = ViewIdentifierBuilder.build(scopeInstance: "adyen.QRCode", postfix: "copyButton")
+        
+        let button = SubmitButton(style: model.style.copyButton)
+        button.title = ADYLocalizedString("adyen.button.copy", localizationParameters)
+        button.addTarget(self, action: #selector(copyCode), for: .touchUpInside)
+        button.accessibilityIdentifier = accessibilityIdentifier
+        
+        return button
+    }()
+    
+    private lazy var logo: NetworkImageView = {
+        let logo = NetworkImageView()
+        logo.widthAnchor.constraint(equalToConstant: 74.0).isActive = true
```

Comment:
I would suggest to keep 74x48 as a local variable, to avoid "magic numbers" in code